client/packages/plugins/backendCommon/generated/ActiveStoresOnSite.ts

```diff
@@ -0,0 +1,4 @@
+// This file was generated by [ts-rs](https://github.com/Aleph-Alpha/ts-rs). Do not edit this file manually.
+import type { Store } from './Store';
```

Comment:
All these type changes due to adding get active stores on site and renaming message to sync_message